mantle/kola/harness.go

```diff
@@ -700,7 +700,9 @@ ExecStart=%s
 				}
 				if Options.SSHOnTestFailure {
 					plog.Errorf("dropping to shell: kolet failed: %v: %s", err, stderr)
-					platform.Manhole(mach)
```

Comment:
Do you think `plog` the error or using `nolint` to not deal with the error is better here?

Reply:
Well, looking `platform.Manhole(m Machine)` the error is being swallowed, I think I'd do:
```
if err := platform.Manhole(mach); err != nil {
    plog.Errorf("failed to get terminal via ssh: %w", err)
}
```
The errors returned by `platform.Manhole` are important.

Reply:
Make sense. I agree with that.